validator/src/lib.rs

```diff
@@ -390,7 +390,17 @@ pub fn main() {
                 .takes_value(false)
                 .help("Skip ledger verification at node bootup"),
         )
-         .get_matches();
+        .arg(
```

Comment:
`solana-validator --cuda ...` is now how we enable CUDA, and an `--opencl` flag should slide in here quite nicely (cc: https://github.com/solana-labs/solana-perf-libs/issues/52)